test/intervals/TestIntervals.jl

```diff
@@ -183,7 +181,9 @@ facts("IntervalCollection") do
 
 
     context("Show") do
-        nullout = open("/dev/null", "w")
+        nullstream = @windows ? "NUL" : "/dev/null"
+        nullout = open(nullstream, "w")
```

Comment:
You're using too old master to use `DevNull`; my patch was merged on Nov 20. (https://github.com/JuliaLang/julia/pull/13758).